backend/src/test/java/quarano/actions/QuarantineEndCheckerTest.java

```diff
@@ -1,20 +1,15 @@
 package quarano.actions;
 
-import static org.assertj.core.api.Assertions.*;
-import static org.mockito.ArgumentCaptor.*;
-import static org.mockito.Mockito.*;
```

Comment:
Correct me if I'm wrong, but I learned `*` imports should be avoided to bypass potential breaking changes of a dependency update.